src/test/kotlin/pl/touk/krush/BookTest.kt

```diff
@@ -44,4 +47,11 @@ class BookTest {
         }
     }
 
+    /**
```

Comment:
\[detekt\] INFO: \[comments/CommentOverPrivateFunction\] Comments for private functions should be avoided\. Prefer giving the function an expressive name\. Split it up in smaller, self\-explaining functions if necessary\.